src/api/api.cc

```diff
@@ -10300,6 +10300,24 @@ extern "C" void V8RecordReplayNewCheckpoint() {
   recordreplay::NewCheckpoint();
 }
 
+// The payload should be the serialized json data for the event.
```

Comment:
Can you add to this comment elaborating what this data and interface is needed for?